src/popup/stores/AccountLoginStore.ts

```diff
@@ -25,20 +27,47 @@ export default class AccountLoginStore {
     );
   }
 
+  /** @params andValidateNetwork: when set to true, disallows being on a network without
```

Comment:
nit: comments should follow this structure. you'll know when VSCode has the highlighting correct.
```
/**
 * Description of the function.
 * @param {boolean} andValidateNetwork Description of param.
 */
```